src/vs/base/parts/quickopen/browser/quickOpenWidget.ts

```diff
@@ -596,8 +600,13 @@ export class QuickOpenWidget extends Disposable implements IModelProvider {
 
 		// Show based on param
 		if (types.isString(param)) {
+			this.withPrefix = true;
 			this.doShowWithPrefix(param);
 		} else {
+			this.withPrefix = false;
+			if (this.options.prefill) {
```

Comment:
Could you instead pass the last value as `options.value` to show()? That will keep the widget separated from the logic in `quickOpenController.ts`.

Reply:
But how can I get the `lastInputValue` into the `quickOpenController`?